api/DeviceLightEvent.json

```diff
@@ -17,14 +17,15 @@
           "firefox": [
             {
               "version_added": "62",
+              "version_removed": "89",
               "flags": [
                 {
                   "type": "preference",
                   "name": "device.sensors.ambientLight.enabled",
                   "value_to_set": "true"
                 }
               ],
-              "notes": "See <a href='https://bugzil.la/1462308'>bug 1462308</a>."
+              "notes": "See <a href='https://bugzil.la/1462308'>bug 1462308</a>, <a href='https://bugzil.la/1701824'>bug 1701824</a>."
```

Comment:
Like the other PR, I'd advise not including the new link or dropping this note outright.